tasks/winbuildscripts/buildwin.bat

```diff
@@ -2,6 +2,13 @@ if not exist c:\mnt\ goto nomntdir
 
 @echo c:\mnt found, continuing
 
+REM
+REM This is a tweak for the in-house CPython build which requests a weird compiler version.
+REM The registry keys will redirect the nonexisting v13.2 compiler to the local VS 2019 toolchain.
+REM
+reg add "HKEY_CURRENT_USER\Software\Microsoft\DevDiv\VCForPython\13.2" /v InstallDir     /t REG_SZ    /d C:\devtools\vstudio\VC\Auxiliary\Build\ /f
+reg add "HKEY_CURRENT_USER\Software\Microsoft\DevDiv\VCForPython\13.2" /v StartMenuItems /t REG_DWORD /d 1                                       /f
+
```

Comment:
I'm also more inclined to have this in the container image.

Reply:
Actually, it is important we don't do this here: if someone runs this `buildwin.bat` file on their local machine and we change compiler registry keys, we could break their compiler setup. This should be set in the container image only.

Reply:
On the contrary, it will build break if the keys are not present on a custom build env.

Do you want they face build break? If so, I can change to docker only.